compiler/next/lib/uast/ASTNode.cpp

```diff
@@ -161,6 +162,8 @@ static void dumpHelper(const ASTNode* ast, int maxIdLen, int depth) {
     printf("%s ", named->name().c_str());
   } else if (const Identifier* ident = ast->toIdentifier()) {
     printf("%s ", ident->name().c_str());
+  } else if (const Comment* comment = ast->toComment()) {
```

Comment:
Could we stop at some threshold so that super long comments don't blow up the dump output?

Reply:
Sure but we need a real uAST dumper at some point anyway. This change is just to make it easy to check the kinds of programs we have been writing in the tests.